docs: add contributor guidelines (#397)
Solaestas/Everglow: README.md

<div align="center">

<img src="https://raw.githubusercontent.com/CycloneClub/Everglow/refs/heads/master/Sources/Everglow/icon.png" alt="Everglow" width="200">

# Everglow

> Everglow is a content-focused Terraria mod in active development. It adds new regions, gameplay systems, equipment, and visual effects for a fresh Terraria experience.

Author: ```suggestion
Everglow is a *freaking awesome* mod for Terraria!.
```
